fpga_interchange/parameter_definitions.py

```diff
@@ -207,16 +207,21 @@ def encode_integer(self, int_value):
             assert self.width is not None
             assert int_value >= 0, int_value
             assert int_value <= (2**self.width), (self.width, int_value)
-            return "{}'b{:b}".format(self.width, int_value)
+            return "{width}'b{value:0{width}b}".format(
+                width=self.width, value=int_value)
         elif self.string_format == ParameterFormat.VERILOG_HEX:
             assert self.width is not None
             assert int_value >= 0, int_value
             assert int_value <= (2**self.width), (self.width, int_value)
-            return "{}'h{:X}".format(self.width, int_value)
+            digits = int(self.width / 4)
```

Comment:
I think technically speaking this should be being rounded up, not down (so something like `(self.width + 3)//4`)

Reply:
You are right, in case un-even widths this would probably truncate the first digit. Fixed.